components/ClaimStatus.tsx

```diff
@@ -2,20 +2,24 @@ import { useTranslation } from 'next-i18next'
 
 import { NextSteps } from './NextSteps'
 import { TextLine } from './TextLine'
+import { TransLine } from './TransLine'
 import { ClaimStatusContent } from '../types/common'
 
 export interface ClaimStatusProps extends ClaimStatusContent {
   loading: boolean
 }
 
-export const ClaimStatus: React.FC<ClaimStatusProps> = ({ loading = false, statusDescription, nextSteps }) => {
+export const ClaimStatus: React.FC<ClaimStatusProps> = ({ loading = false, heading, summary, nextSteps }) => {
   const { t } = useTranslation(['common', 'claim-status'])
 
   return (
     <div className="claim-status">
       <h2>{t('claim-status.title')}</h2>
       <div className="pending-status">
-        <TextLine loading={loading} text={t(statusDescription)} />
+        <TextLine loading={loading} text={t(heading)} />
+      </div>
+      <div className="summary">
+        <TransLine i18nKey={summary.i18nKey} links={summary.links} />
```

Comment:
@lomky I just tossed the heading and summary in here like this for now and assumed that you'd rework how they get rendered.

Reply:
perfect